test/matrix_api/map_copy_extension_test.dart

```diff
@@ -30,11 +30,11 @@ void main() {
           'list': [1, 2],
         },
       };
-      final copy = original.copy();
+      final copy = copyMap(original);
       original['child']['attr'] = 'raccoon';
-      expect(copy['child']['attr'], 'bunny');
+      expect((copy['child'] as Map)['attr'], 'bunny');
```

Comment:
Do we need this cast now or is this just an additional check? I can't quite tell what we are doing different, that makes this necessary now? Was it because the old map was `<String, dynamic>` and this is `<String, Object?>` now?

Reply:
We need it as the new copymap method now returns a `Map<String, Object>` and not `Map<String, dynamic>` anymore so `copy['child']` is a `Object?` and we need to cast it before we can access 'attr'

Reply:
I see, thanks for explaining! :)